appveyor.yml

```diff
@@ -1,5 +1,5 @@
 init:
-  - git config --global core.autocrlf true
+  - git config --global core.autocrlf input
```

Comment:
@dougbu I am fully aware of Windows developer resistance if you try to get rid of their carriage return. :sweat_smile: That is why adding SourceLink to your project doesn't change your local development at all. SourceLink only runs on the CI builds. This is documented in the readme at the end of [this section](https://github.com/ctaggart/SourceLink#dotnet-sourcelink). The next section about dotnet-sourcelink-git explains the line endings and then there is a dedicated [wiki page on line endings](https://github.com/ctaggart/SourceLink/wiki/Line-Endings). You need to not change the line ending on the build server to enable source linking. This only affects Windows CI builds. What are you using for the official builds? Team City? Do you control the `git clone` command? Aside, it will work without this settings, since I have SourceLink autofix the line endings by default. It will just be slow.